### ISSUE TITLE
wrong magic find calculation?

### ISSUE THREAD

**Issue by Parkusisafk** [carreb/meterthing#5]
base rate * (1+mf/100)

**Comment by t4ttc** [comment]
yes that is the default magic find calculation, however meterthing also factors in the RNG meter's increase in weight in its calculation. as you can see in `index.js:146`, the formula you provided is used, with the base rate being swapped out with the rate after the increase applied by the rng meter.
```js
selectedItemWeight = selectedItemWeight * (1 + (pog.magic_find / 100));
```

the only difference being the base rate is increased by the RNG meter, which increases as described by this formula in `index.js:110`
```js
selectedItemWeight = selectedItemWeight * (1 + Math.min((2 * lastXP / droptables[pog.rng_meter_selected_table][pog.rng_meter_selected_item].required_xp), 2));
```
